src/mito2/src/worker/handle_alter.rs

```diff
@@ -167,10 +178,89 @@ impl<S> RegionWorkerLoop<S> {
                         current_options.ttl = Some(new_ttl);
                     }
                 }
+                ChangeOption::Twsc(key, value) => {
+                    let parse_usize_with_default = |key: &str, value: &str, default: usize| {
```

Comment:
Prefer to extract this match clause into an independent function such as  `changeTwcsOption(key, value)`.

Reply:
thanks implemented. the new update is coming alone nicely.